Supervisor comments (#54)
chhoumann/p8: report/src/problem_analysis/problems_statement.tex

To resolve these issues, the stakeholders proposed a system defined by several user stories detailed in Appendix \ref{appendix:user_stories}. 
To examine whether such a system would solve the stakeholders' issues, a hifi prototype was developed using Figma\cite{Figma} by following methods from user-centred design.
The prototype was tested using an approach to summative usability testing defined by \citeauthor{lazar2005web}~\cite{lazar2005web}. 
The result of this test was that, while many issues regarding usability could be resolved by a system based on the prototype, some still remain. 
One issue remaining is that meeting rooms might still be unused due to reliance on user interactions for ending the meeting early.
It was concluded that a solution to Mariendal ITs problems should provide functionality for minimizing reliance on user interaction.
To alleviate such issues, a technology that can automatically identify when a room is unused could be deployed. 
Techniques detecting whether people or objects are within specific boundaries is also known as \textit{presence detection}.

> Further exploring approaches for presence detection could be used to develop a method for automatically ending meetings when participants leave the room.

Reviewer: ```suggestion
By further exploring approaches for presence detection, it could be possible to develop a method that automatically ends meetings when participants leave the room.
```
